Calibration/LumiAlCaRecoProducers/python/ALCARECOAlCaPCCZeroBiasFromRECO_Output_cff.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+# AlCaReco for track based calibration using MinBias events
+OutALCARECOAlCaPCCZeroBiasFromRECO_noDrop = cms.PSet(
+    SelectEvents = cms.untracked.PSet(
+        SelectEvents = cms.vstring('pathALCARECOAlCaPCCZeroBiasFromRECO')
+    ),
+    outputCommands = cms.untracked.vstring(
+        'keep *_*alcaPCCProducer*_*_*') 
```

Comment:
do any of these get added to the standard workflows?
We should avoid as much as possible wildcards in the module instance names to avoid making unnecessary modules to be run in the unscheduled setup.
This seems to have specific enough name that could probably be left as is.